### ISSUE TITLE
Not able use keyboard to select table. (X508 compliance)

### ISSUE THREAD

**Issue by KalpanaPagariya** [ant-design/ant-design#5652]
### Version
2.9.0

### Environment
Chrome

### Reproduction link
[http://codepen.io/kalpanapagariya/pen/pemPEx](http://codepen.io/kalpanapagariya/pen/pemPEx)

### Steps to reproduce
Can try out to select table record using keyboard.

### What is expected?
Using keyboard user should be able to select record, click  button. (Accessibility ARIA)

### What is actually happening?
Selectiin not working using keyboard

**Comment by yesmeck** [comment]
Hello @KalpanaPagariya , rows can be selected by <kbd>tab</kbd> and <kbd>space</kbd>.

**Comment by KalpanaPagariya** [comment]
Thanks for reply. I was trying tab and Enter
